k8s/airflow-operator/README.md

```diff
@@ -172,6 +172,10 @@ Use `envsubst` to expand the template. We recommend that you save the expanded
 manifest file for future updates to the application.
 
 ```shell
+# Install envsubst
+apt-get install -y gettext-base
```

Comment:
It will works only for debian based systems.
In my eyes best way is just provide this as requirement for user environment.
Like this: https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/cert-manager#prerequisites

Reply:
Nice point. But I can't find the requirement in the link you sent me. Can you please send me exactly where is located the comment you have suggested? Thanks

Reply:
Fixed adding the instruction to "Setup Command line tools" section.